src/actions/file-search.js

```diff
@@ -86,7 +86,7 @@ export function searchContents(query: string, editor: Object) {
     const matches = await getMatches(
       query,
       selectedSource.get("text"),
-      _modifiers
+      modifiers
     );
```

Comment:
is it safe to pass an immutable object here? we might not have good coverage here so please test manually.

Reply:
This shouldn't be here iirc, will check.